esrc/rescueoverview.php

```diff
@@ -473,4 +483,86 @@ function displayNotes($row, $isCoord = 0, $isSARAgent = 0)
 		}
 	}
 }
+
+/**
```

Comment:
In general: Would it be a good separation of code/concern to move these functions to the killboard check class? Not sure if this code should be part of this page itself.